lib/manifestManager/index.js

```diff
@@ -0,0 +1,28 @@
+export class ManifestManager {
+  static $inject = ['MANIFEST_PATH'];
+  static $singleton = true;
+
+  constructor(manifestPath) {
+    if(!manifestPath) {
+      throw 'Can\'t init ManifestManger instance without passing manifestPath as a param.';
+    }
+    this.manifestPath = manifestPath;
```

Comment:
I'm just not 100% sure we're going to loadManifest in manifest construnctor always. In case we are, i'll just pass it down directly as a argument to loadManifest from constructor.